moveit_ros/planning/kinematics_plugin_loader/src/kinematics_plugin_loader.cpp

```diff
@@ -68,16 +69,17 @@ class KinematicsPluginLoader::KinematicsLoaderImpl
    * \param search_res
    * \param iksolver_to_tip_links - a map between each ik solver and a vector of custom-specified tip link(s)
    */
-  KinematicsLoaderImpl(const rclcpp::Node::SharedPtr& node, const std::string& robot_description,
+  KinematicsLoaderImpl(node_interface::NodeInterfaceSharedPtr node_interface, const std::string& robot_description,
                        const std::map<std::string, std::vector<std::string>>& possible_kinematics_solvers,
                        const std::map<std::string, std::vector<double>>& search_res,
                        const std::map<std::string, std::vector<std::string>>& iksolver_to_tip_links)
-    : node_(node)
+    : node_interface_{ std::move(node_interface) }
     , robot_description_(robot_description)
     , possible_kinematics_solvers_(possible_kinematics_solvers)
     , search_res_(search_res)
     , iksolver_to_tip_links_(iksolver_to_tip_links)
   {
+//    node_interface_ = node_interface;
```

Comment:
remove commented out code, with the std::move this would be UB if you uncommented it

Reply:
Removed